lib/prosperworks/User.rb

```diff
@@ -0,0 +1,12 @@
+module ProsperWorks
+  class User < BaseEntity
+
+    attr_accessor :id,
```

Comment:
Ah, and `Base` already declares: `attr_accessor :id`, so you can remove this line